src/app/character-tile/CharacterTile.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import clsx from 'clsx';
+import { DimStore } from '../inventory/store-types';
+import { AppIcon, powerActionIcon } from '../shell/icons';
+import './CharacterTile.scss';
+
+const CharacterEmblem = ({ store }: { store: DimStore }) => (
+  <div
+    className={clsx('emblem', { vault: store.isVault })}
+    style={{ backgroundImage: `url("${store.icon}")` }}
+  />
+);
+
+export const CharacterTile = ({ store }: { store: DimStore }) => (
+  <div className="character-tile">
+    <div className="background" style={{ backgroundImage: `url("${store.background}")` }} />
+    <CharacterEmblem store={store} />
+    <div className="character-text">
+      <div className="top">
+        <div className="class">{store.className}</div>
+        {!store.isVault && (
+          <div className="powerLevel">
+            <AppIcon icon={powerActionIcon} />
+            {store.powerLevel}
+          </div>
+        )}
+      </div>
+      {!store.isVault && (
+        <div className="bottom">
+          <div className="race-gender">{store.genderRace}</div>
+          {store.isDestiny1() && store.level < 40 && <div className="level">{store.level}</div>}
```

Comment:
Not saying you should change it now, but I wonder if we could put some more interesting info in here than `store.genderRace`.

Reply:
for sure! just copy + pasted from the existing file.